internal/handlers/pg/pgdb/pool.go

```diff
@@ -262,15 +262,24 @@ func (pgPool *Pool) SchemaStats(ctx context.Context, schema, collection string)
 }
 
 // SetDocumentByID sets a document by its ID.
-func (pgPool *Pool) SetDocumentByID(ctx context.Context, db, collection string, id any, doc *types.Document) (int64, error) {
+func (pgPool *Pool) SetDocumentByID(
+	ctx context.Context, db, collection, comment string, id any, doc *types.Document,
```

Comment:
Maybe we can change with signature and use `sp SQLParam` instead of db, collection, and comment? So then, there are fewer parameters for this method.

Reply:
Yeah. I was thinking about that, but then you have an import loop. We could extract SQLParam to the seperate package or leave it as it is.

Reply:
Why an import loop? There are two packages - `pg` and `pgdb`. If I understand everything correctly, `SQLParam` is part of `pgdb`, `pg` imports `pgdb`, but `pgdb` doesn't impoty `pg` :)

Reply:
My bad :)